config/webpack.chart-renderer.config.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-env node*/
+/*eslint import/no-nodejs-modules:0 */
+const baseConfig = require('../webpack.config');
+
+const config = {
+  ...baseConfig,
+
+  target: 'node',
+  entry: {
+    chartRendererConfig: 'app/components/charts/backendStyles',
+  },
+
+  optimization: {},
+
+  output: {
+    ...baseConfig.output,
+    libraryTarget: 'commonjs',
+  },
+};
+
+module.exports = config;
```

Comment:
Going to try and pare the output file down a bit. Right now it's a whopping 1.25mb, which is absolutely _way_ too large.